contracts/NFTMarket.sol

```diff
@@ -154,18 +159,18 @@ contract NFTMarket is IERC721Receiver, ReentrancyGuard {
         // There must be at least 1 recipient.
         if (_recipients.length <= 0) revert NoRecipients();
 
-        // Add up all `SaleRecipeint.percent` allotments to make sure they sum to no more than 100%.
+        // Storage for recipient percentage validation.
         uint256 saleRecipientsPercentTotal = 0;
 
-        // Validate that recipients add up to no more than 100%.
+        // Validate that all `SaleRecipeint.percent` sum to no more than 10000 (100%).
         for (uint256 i = 0; i < _recipients.length; i++) {
             // Each recipient's percent must be greater than 0.
             if (_recipients[i].percent <= 0) revert RecipientPercentZero();
 
             // Add to the total percents.
             saleRecipientsPercentTotal = saleRecipientsPercentTotal + _recipients[i].percent;
 
-            // The sum of percent values must not exceed 10000.
+            // The sum of percent values must not exceed 10000 (100%).
             if (saleRecipientsPercentTotal > 10000) revert PercentExceeded();
```

Comment:
let's extract 10000 out into a meaningful constant?

Reply:
@odd-amphora Would `const ONE_HUNDRED_PERCENT = 10000` be a good choice? 
Linter complains that "Variable name must be in mixedCase [var-name-mixedcase]."